### PR TITLE
Format some tests

### PR DESCRIPTION
**Pull request by nnethercote**
There are more directories under `tests/` still to do, but this is enough for one PR.

r? @GuillaumeGomez

### CODE REVIEW

**Comment by nnethercote** [comment]
This touches a lot of files and is conflict prone, so:

@bors rollup=never p=1

File: tests/run-make/reproducible-build-2/reproducible-build.rs [hunk 1]
```diff
@@ -94,13 +89,15 @@ fn main() {
...
 
-    with_fn_once_adapter(|_:i32| { });
+    with_fn_once_adapter(|_: i32| {});
 
     reproducible_build_aux::regular_fn(STATIC);
     reproducible_build_aux::generic_fn::<u32, char>();
     reproducible_build_aux::generic_fn::<char, Struct<u32, u64>>();
-    reproducible_build_aux::generic_fn::<Struct<u64, u32>,
-                                         reproducible_build_aux::Struct<u32, u64>>();
+    #[rustfmt::skip] // rustfmt formats this as longer than 100 chars, which tidy doesn't like!
```
Comment (lqd) [inline tests/run-make/reproducible-build-2/reproducible-build.rs:97]: The `// ignore-tidy-linelength` directive may work to avoid doing this

Comment (nnethercote) [reply tests/run-make/reproducible-build-2/reproducible-build.rs]: Ok, I have switched to that.

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
r=me once it's ready for you (and yeay, thanks a lot!).

**Comment by fmease** [comment]
I don't want my rollup of 4 to jump ahead of this PR (that'd be kinda mean ^^).
@bors p=5

**Comment by nnethercote** [comment]
I fixed the windows test failure.

@bors r=GuillaumeGomez

**Comment by jieyouxu** [comment]
(We should update `.git-blame-ignore-revs` to include these commits after this PR is merged)

**Pull request merged**